lib/start.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
```

Comment:
In other plugins `lib` folder in gitignore and compile with `ris all compile` command